include/TrackContainerView.h

```diff
@@ -148,6 +148,8 @@ class TrackContainerView : public QWidget, public ModelView,
 		return "trackcontainerview";
 	}
 
+	int calcTotalHeightOfTracks();
+
```

Comment:
I'd just go with `totalHeightOfTracks()` but that's optional. There's a redundant newline here though.